### PR TITLE
Les 290 1

### PR DESCRIPTION
**Pull request by IgorTanov**
created a button in the header to change the theme (dark-light)

### CODE REVIEW

**Review by Terapool — APPROVED** [review]

**Comment by Evgeniy121212** [comment]
image

**Review by Evgeniy121212 — COMMENTED** [review]
image

Please, remove that submodule - we don't needed that into a project :)

**Comment by IgorTanov** [comment]
remove lessoner_project_frontend and rework HEader.tsx

File: src/hooks/useTheme/useTheme.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useLayoutEffect, useState } from "react";
+
+export const useTheme=()=>{
+    const [theme,setTheme]=useState(localStorage.getItem("theme") || "dark")
+    useLayoutEffect(()=>{
+        document.documentElement.setAttribute("data-theme",theme)
+        localStorage.setItem("theme",theme)
```
Comment (Evgeniy121212) [inline src/hooks/useTheme/useTheme.tsx:7]: let's avoid `document.documentElement.setAttribute("data-theme",theme)`

File: src/index.css [hunk 2]
```diff
@@ -55,15 +53,33 @@ code {
...
+
+
+html[data-theme="dark"] {
+  --text-color: #e4e1e6;
+  --bg-color: #1b1b1f;
+}
+html[data-theme="light"] {
+  --text-color: #1b1b1f;
+  --bg-color: #e4e1e6;
+}
```
Comment (Evgeniy121212) [inline src/index.css:81]: let's don't use `html[data-theme="dark"]` and `html[data-theme="light"]`

File: src/hooks/useTheme/useTheme.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { useLayoutEffect, useState } from "react";
+
+export const useTheme=()=>{
+    const [theme,setTheme]=useState(localStorage.getItem("theme") || "dark")
```
Comment (Evgeniy121212) [inline src/hooks/useTheme/useTheme.tsx:4]: let's move "dark" and "light" to the constants

**Review by valer-sky — APPROVED** [review]

**Pull request closed without merging**